gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/assertions/Assertions.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package assertions;
+
+import jakarta.ws.rs.core.Response;
+
+public class Assertions extends org.assertj.core.api.Assertions {
```

Comment:
I would choose a very (very very) explicit name here: `MAPIAssertions` or `RestAssertions`
My fear here is in few month we will not make the difference between this class and the original `assertj` one.
It will not disturb the readability of the code because it is only about changing the static import
wdyt ?